### PR TITLE
New configuration for build the image

### PR DESCRIPTION
**Pull request by rafaeltovargarrido**
adding a new **Dockerfile** for the construction of the app.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -22,4 +22,7 @@ APP_SECRET=3f33e29c19d9d24f0dccb90a3f84db04
 
 ###> oat-sa/bundle-lti1p3 ###
 LTI1P3_SERVICE_ENCRYPTION_KEY=a725e29c19dabee77hdccb90a3f84db04
-###< oat-sa/bundle-lti1p3 ###
\ No newline at end of file
+###< oat-sa/bundle-lti1p3 ###
+
+###> Cache ###
+REDIS_CACHE_DSN=redis://demo_lti1p3_redis:6379
```
Comment (ekkinox) [inline .env]: please add new line

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            echo 'opcache.load_comments=1'; \
+            echo 'opcache.validate_timestamps=0'; \
+            echo 'realpath_cache_size=4096K'; \
+            echo 'realpath_cache_ttl=600'; \
+        } >> /usr/local/etc/php/conf.d/docker-php-ext-opcache.ini \
+        ; \
+        mv /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini; \
+        rm /usr/local/etc/php-fpm.d/www* \
+        ; \
+        chown -R www-data: /var/www/html/
```
Comment (ekkinox) [inline Dockerfile]: please add new line

**Review by ekkinox — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (rafaeltovargarrido) [reply .env]: Done

File: Dockerfile (hunk 2 above)
Comment (rafaeltovargarrido) [reply Dockerfile]: Done

**Review by ekkinox — APPROVED** [review]

**Pull request merged**